### PR TITLE
Feature/patient medical report

### PR DESCRIPTION
**Pull request by juliaorduno**
### What does this PR do?
- Add Patient's Medical Report view

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Scout (e.g. improves readability or does some small fixes without causing **ANY** impact)

### Where should the reviewer start? Any background context you want to provide?
:checkered_flag:
Go to http://dev.health-mx.com:8000/medic/appointments and cllick in any row or go to http://dev.health-mx.com:8000/medic/patient-profile

#### Checklist
- [x] I added at least 2 reviewers
- [ ] I have added necessary documentation (if appropriate)
- [ ] I did review at least 2 **existing pull requests** before submitting mine.
- [ ] I have added tests proving that my fix is effective and/or the feature works
- [ ] In case this is a `breaking change`, I have already warned backend and/or frontend
- [ ] I already added the pull request link to related Issue
- [ ] I have cleaned up at least one file before checking in 
- [ ] I handled nicely the exceptions in possible scenarios where my changes could make the app crash

### CODE REVIEW

File: __test__/__unitTesting__/components/pages/Medic/MedicalReport/Tabs.test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  endDate: '11 Nov 2017',
+  medication: ['Paracetamol', 'Diclofenaco sódico', 'Celebrex'],
+}]
+const consultations = [{
+  date: '12 Jul, 2018',
+  area: 'Medicina familiar',
+  doctor: 'Luis Alfredo Ramírez',
+  status: 'Regular',
+}]
+const tabsValues = [
```
Comment (MikeSagnelli) [inline __test__/__unitTesting__/components/pages/Medic/MedicalReport/Tabs.test.js]: Please use __mocks__ and testData folders for this type of data. Tests files should only contain tests

File: components/pages/Medic/MedicalReport/Tabs.jsx [hunk 2]
```diff
@@ -0,0 +1,259 @@
+import React from 'react'
+import { Tab, Grid, List, Table } from 'semantic-ui-react'
+import styled from 'styled-components'
+import media from 'styled-media-query'
+import ValueItem from '../../../shared/ValueItem'
+import CsvList from './CsvList'
+
+const TabMedicalReport = ({ beneficiary, consultations, studies, treatments, callbackfn }) => {
+
+  const panes = [
```
Comment (MikeSagnelli) [inline components/pages/Medic/MedicalReport/Tabs.jsx]: Please abstract this to another file, no render methods should exist other than the file's component render method

File: components/shared/ValueItem/index.jsx [hunk 3]
```diff
@@ -2,9 +2,9 @@ import React from 'react'
 import styled from 'styled-components'
 
 const ValueItem = ({ keyName, value }) => (
-  <React.Fragment>
+  <div>
```
Comment (MikeSagnelli) [inline components/shared/ValueItem/index.jsx]: Never use empty `<div>` tags. Please only use these if a className will be added, if not, use React.Fragment.

**Comment by alfonsoirai** [comment]
The **play** icon in the "Iniciar consulta" label is missing.

**Pull request merged**